### PR TITLE
REF [FEATURE] [54790] Create study wrong answer question screen

### PR DESCRIPTION
**Pull request by minhnn-1279**
[Feature #54790: Wrong Answer Screen](https://edu-redmine.sun-asterisk.vn/issues/54790)

What:
- Create UI for Wrong Answer Screen
- Fetch data and display
- Handle event

image
image

### CODE REVIEW

File: app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/utils/base/BaseFragment.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.nguyennhatminh614.motobikedriverlicenseapp.utils.base
 
 import android.os.Bundle
+import android.util.Log
```
Comment (hungld-1302) [inline app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/utils/base/BaseFragment.kt]: remove

File: app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/utils/dialog/LoadingDialog.kt [hunk 2]
```diff
@@ -2,6 +2,7 @@ package com.nguyennhatminh614.motobikedriverlicenseapp.utils.dialog
 
 import android.app.AlertDialog
 import android.content.Context
+import android.util.Log
```
Comment (hungld-1302) [inline app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/utils/dialog/LoadingDialog.kt]: .

File: app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/utils/dialog/LoadingDialog.kt [hunk 3]
```diff
@@ -14,16 +15,19 @@ object LoadingDialog {
...
+        }
 
         dialog?.show()
         dialog?.window?.attributes = windowLayoutParams
     }
 
     fun hideLoadingDialog() {
-        dialog?.hide()
+        dialog?.dismiss()
+        dialog = null
```
Comment (hungld-1302) [inline app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/utils/dialog/LoadingDialog.kt]: remove

File: app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/screen/wronganswer/QuestionWrongAnswerFragment.kt [hunk 4]
```diff
@@ -0,0 +1,136 @@
+package com.nguyennhatminh614.motobikedriverlicenseapp.screen.wronganswer
+
+import android.util.Log
```
Comment (hungld-1302) [inline app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/screen/wronganswer/QuestionWrongAnswerFragment.kt]: .

File: app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/data/model/WrongAnswerObject.kt [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import android.os.Parcelable
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.nguyennhatminh614.motobikedriverlicenseapp.data.model.WrongAnswerObject.Companion.WRONG_ANSWER_TABLE
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+@Entity(tableName = WRONG_ANSWER_TABLE)
+data class WrongAnswerObject(
+    @PrimaryKey val questionsID: Int,
```
Comment (vietnh-1109) [inline app/src/main/java/com/nguyennhatminh614/motobikedriverlicenseapp/data/model/WrongAnswerObject.kt:12]:
```suggestion
    @PrimaryKey 
    val questionsID: Int,
```

**Pull request merged**